test/test_dtype.py

```diff
@@ -3,28 +3,69 @@
 from tinygrad.helpers import getenv, DType, DEBUG
 from tinygrad.lazy import Device
 from tinygrad.tensor import Tensor, dtypes
+from typing import List, Optional
 from extra.utils import OSX, temp
+import copy
+
 
 def _test_to_np(a:Tensor, np_dtype, target):
   print(a)
```

Comment:
This should probably be gated behind `DEBUG=2` as well, it wasn't before but it makes CI a *tiny tiny tiny* bit faster.

Reply:
Done